blocks/contacts/contacts.css

```diff
@@ -0,0 +1,9 @@
+.contacts{
+  width:100% ;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  position: relative;
```

Comment:
Why do you need `position: relative;` here? 🤔

Reply:
@naumch1k I have removed this unwanted element